jupyterhub/auth.py

```diff
@@ -260,39 +294,74 @@ def my_hook(authenticator, handler, authentication):
 
     def __init__(self, **kwargs):
         super().__init__(**kwargs)
-        for method_name in (
-            'check_whitelist',
-            'check_blacklist',
-            'check_group_whitelist',
+        self._init_deprecated_methods()
+
+    def _init_deprecated_methods(self):
+        # TODO: properly handle deprecated signature *and* name
+        # with correct subclass override priority!
```

Comment:
```suggestion
        # handles deprecated signature *and* name
        # with correct subclass override priority!
```
not a todo anymore :tada:

Reply:
Thanks!